### PR TITLE
Improve documentation and formatting

### PR DESCRIPTION
**Pull request by davesnx**
https://github.com/draftbit/bs-storybook/blob/davesnx/Update-readme/README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +34,17 @@ configure(() => {
 }, module);
 ```
 
-Note that in the above example, we're assuming the convention of each module containing a function as the `default` export. We'll account for that when writing our stories in the next section.
+or if you are using Storybook v6.
```
Comment (peterpme) [inline README.md:37]: should we support v6 moving forward?

File: README.md [hunk 2]
```diff
@@ -41,32 +56,66 @@ open BsStorybook.Story;
...
-   );
+->add("Chapter I", () => <span> {React.string("Hello bs-storybook!")} </span>);
 ```
 
 Storybook uses a reference to the `module` global provided by webpack to facilitate hot-reloading. We'll access that via the `[%bs.raw]` decorator.
 
 ## The Actions Addon
 
 The action addon's API is essentially unchanged from its JS implementation:
+> Make sure that you have @storybook/addon-actions in the config
```
Comment (peterpme) [inline README.md:67]:
```suggestion
> Make sure that you have `@storybook/addon-actions` in the config
```

File: README.md [hunk 3]
```diff
@@ -128,16 +177,47 @@ let greeting = Knobs.select(~label="Greeting", ~options, ~defaultValue=options.o
...
+let obj = Knobs.object_(~label="User", ~defaultValue={"color": "grey"}, ());
+```
+
+### Js.Dict
+> https://bucklescript.github.io/bucklescript/api/Js.Dict.html
+
+```reason
+let options =
+  Js.Dict.fromArray([|
+		("Red", "red"),
```
Comment (peterpme) [inline README.md:198]: tabs?

Comment (davesnx) [reply README.md:198]: Let me add a editorconfig as well 👍

File: README.md (hunk 1 above)
Comment (davesnx) [reply README.md:37]: v5 and v6 are supported, right now there's no breaking change or reason to not support v5.

**Pull request merged**